main.py

```diff
@@ -603,6 +603,15 @@ def handle_flows_install_delete(self, event):
             return
 
         force = bool(event.content.get("force", False))
+        if not flow_dict["flows"]:
+            log.error(f"Error, empty list of flows recieved. {flow_dict}")
+            return
+        else:
+            log.info(
+                f"Received event with {len(flow_dict['flows'])} flows"
+                f" with force={force}."
+            )
+
```

Comment:
I just realized that if via event the dpid isn't found it'll crash too. 

Via events we're only supporting to send to a single dpid, so if the dpid isn't found it should also log as error and not crash. If you can cover this too.

Reply:
It is covered in line 589

Reply:
The `dpid` key yes, but its value it's not yet. Check out this [line here](https://github.com/kytos-ng/flow_manager/blob/log_flows/main.py#L621), with a non existing dpid it would result in a `[None]` which would lead to problems.

Reply:
Oh, got it. Added statement to check if a switch was found.